pkg/apis/core/validation/validation.go

```diff
@@ -5852,19 +5844,6 @@ func validateContainerResize(newRequirements, oldRequirements *core.ResourceRequ
 	return allErrs
 }
 
-func resourcesRemoved(resourceList, oldResourceList core.ResourceList) bool {
-	if len(oldResourceList) > len(resourceList) {
-		return true
-	}
-	for name := range oldResourceList {
-		if _, ok := resourceList[name]; !ok {
-			return true
-		}
-	}
-
-	return false
-}
-
```

Comment:
this is changing a behavior in a graduated beta api, should you not first amend this change to the corresponding KEP instead of trying to get the change directly? or is this PR just asking for feedback?

Reply:
oh, I See @tallclair already commented here https://github.com/kubernetes/kubernetes/pull/127143#issuecomment-2702463416 ,